app/views/warehouse_reports/disabilities/show.haml

```diff
@@ -16,8 +16,7 @@
 
   .row(style="padding-top:1em")
     .col-sm-12.text-right
-      - # FIXME replace permit! with a whitelist and remove from brakeman.ignore
-      = link_to params.permit!.merge(format: :xlsx), class: 'btn btn-secondary' do
+      = link_to({format: :xlsx}, {class: 'btn btn-secondary'}) do
```

Comment:
This will also lose other filter options.

Reply:
There are no filter options on the an enrolled clients with disabilities report.